src/prefect/settings.py

```diff
@@ -1458,6 +1458,12 @@ def default_cloud_ui_url(settings, value):
 The maximum number of scheduled runs to create for a deployment.
 """
 
+PREFECT_DEPLOYMENT_CONCURRENCY_SLOT_WAIT_SECONDS = Setting(float, default=0.0)
```

Comment:
why is the default 0?

Reply:
We felt like latency was enough of a wait to avoid looping thru the same rescheduled run while at the same time not have unnecessary waiting to by runs that can't grab a slot.

Reply:
Let's increase to 30 to match the Cloud PR.